src/lib/pluginUtils.ts

```diff
@@ -102,11 +102,8 @@ export function pluginName(plugin: IPluginState|IContractInfo, fallback?: string
     name = rewarderContractName(plugin as IContributionRewardExtState);
 
     if (!name) {
-      name = "ContributionRewardExt";
-    } else {
-      // add spaces before capital letters to approximate a human-readable title
-      name = `${plugin.name[0]}${plugin.name.slice(1).replace(/([A-Z])/g, " $1")}`;
-    }
+      name = PLUGIN_NAMES["ContributionRewardExt"];
```

Comment:
should it just be here PLUGIN_NAMES[plugin.name as keyof typeof PLUGIN_NAMES]; ?

Reply:
changed to `name = plugin.name`